modules/migration-about-migration-hooks.adoc

```diff
@@ -1,8 +1,9 @@
 // Module included in the following assemblies:
-// migration/migrating_3_4/migrating-application-workloads-3-4.adoc
-// migration/migrating_4_1_4/migrating-application-workloads-4-1-4.adoc
-// migration/migrating_4_2_4/migrating-application-workloads-4-2-4.adoc
-= Understanding migration hooks
+// * migration/migrating_3_4/migrating-application-workloads-3-4.adoc
+// * migration/migrating_4_1_4/migrating-application-workloads-4-1-4.adoc
+// * migration/migrating_4_2_4/migrating-application-workloads-4-2-4.adoc
+[id='migration-about-migration-hooks_{context}']
```

Comment:
Same in this file about reverting the file rename and anchor, and fixing the assembly list.